src/steps/01-AcquisitionPackageDetails/ContactInfo.vue

```diff
@@ -106,6 +106,19 @@
             $validators.required('Please enter your title.')
           ]"
         />
+        <ATATPhoneInput
+          label="Your phone number"
+          id="ContactPhone"
+          :class="{ 'mb-10': selectedRole === 'CIVILIAN' }"
+          :value.sync="selectedPhoneNumber"
+          :country.sync="selectedPhoneCountry"
+          :extension.sync="phoneExtension"
+          :rules="[
+            $validators.isPhoneNumberValid(
+              this.selectedPhoneCountry
+            ),
```

Comment:
- [ ] the prop `  @Prop({ default: true }) private phoneExtension!: boolean;` in the ATATPhoneInput.vue is not being used here. Is it necessary?